pkg/resources/syslogng/configcheck.go

```diff
@@ -48,6 +48,16 @@ func (r *Reconciler) configHash() (string, error) {
 	return fmt.Sprintf("%x", hasher.Sum32()), nil
 }
 
+func (r *Reconciler) CheckForObjectExistence(ctx context.Context, object client.Object) (*ConfigCheckResult, error) {
```

Comment:
The caller (`func Reconcile`) seems to be responsible for returning a `RequeueAfter <time>`, so I opted for just returning a `not-Ready` result, which will make it try again in a minute. Is it acceptable, or should we change the time?

Reply:
1 minute looks a bit long to me, is there any way to reduce this delay for this use case?

Reply:
That is just a fail-safe setting, it shouldn't take a minute. The controller watches the secret it creates, so as soon as the secret is created a reconcile will be triggered. We could even return with a reconcile result:true there, but the 1 minute shouldn't hurt either.

Reply:
@OverOrion however the notready result won't take effect currently as you return an error as well. The end result is going to be the same, so I'm not insisting on changing this, but in case the object has simply not yet been received through the watch, then it looks cleaner to me to return without an error.

Reply:
You are right, changed it accordingly, thanks for it!